code/modules/clothing/shoes/_shoes.dm

```diff
@@ -115,9 +115,10 @@
  * *
  * * state: SHOES_UNTIED, SHOES_TIED, or SHOES_KNOTTED, depending on what you want them to become
  * * user: used to check to see if we're the ones unknotting our own laces
+ * * force_lacing: if TRUE, ignores can_be_tied
  */
-/obj/item/clothing/shoes/proc/adjust_laces(state, mob/user)
-	if(!can_be_tied)
+/obj/item/clothing/shoes/proc/adjust_laces(state, mob/user, force_lacing)
```

Comment:
```suggestion
/obj/item/clothing/shoes/proc/adjust_laces(state, mob/user, force_lacing = FALSE)
```

if we want it to be a boolean then let's type it that way so it's doubly extra explicit instead of being `null` for any given weird reason